### PR TITLE
code changes

### PR DESCRIPTION
**Pull request by dorin-iancu**

### CODE REVIEW

**Review by alyn509 — APPROVED** [review]

File: esdt-safe/src/from_sovereign/token_mapping.rs [hunk 1]
```diff
@@ -28,28 +46,31 @@ pub trait TokenMappingModule:
...
                 token_type,
                 issue_cost,
                 token_display_name,
                 token_ticker,
                 num_decimals,
-            ),
+            }),
         }
 
+        // !!! Unreachable code !!!
```
Comment (axenteoctavian) [inline esdt-safe/src/from_sovereign/token_mapping.rs:73]: Really? This worked before :thinking:

Comment (andreiblt1304) [reply esdt-safe/src/from_sovereign/token_mapping.rs:73]: I was thinking the same thing

Comment (andreiblt1304) [reply esdt-safe/src/from_sovereign/token_mapping.rs:73]: does it really matter if we will probably won't use `esdt-safe`?

Comment (axenteoctavian) [reply esdt-safe/src/from_sovereign/token_mapping.rs:73]: Currently this contact is used for public deployment. Hopefully soon we'll move to enshrine-esdt.

**Review by axenteoctavian — APPROVED** [review]

File: esdt-safe/src/from_sovereign/token_mapping.rs (hunk 1 above)
Comment (dorin-iancu) [reply esdt-safe/src/from_sovereign/token_mapping.rs:73]: Yep. Issue-ing a token kills current contract execution.

**Pull request merged**